pkg/batchscheduler/interface/interface.go

```diff
@@ -25,4 +25,5 @@ type BatchScheduler interface {
 
 	ShouldSchedule(app *v1beta2.SparkApplication) bool
 	DoBatchSchedulingOnSubmission(app *v1beta2.SparkApplication) error
+	CleanupOnCompleted(app *v1beta2.SparkApplication) error
```

Comment:
To be consistent, let's rename this to `CleanupOnCompletion`.

Reply:
Done